src/stmem.cc

```diff
@@ -376,3 +382,31 @@ mem_hdr::getNodes() const
     return nodes;
 }
 
+void
+mem_hdr::setIdleness(const bool idle)
+{
+    assert(idle != isIdle);
+    isIdle = idle;
+    if (isIdle) {
+        IdleNodes += size();
+    } else {
+        assert(IdleNodes >= size());
+        IdleNodes -= size();
+    }
```

Comment:
If possible without heroic efforts, please add IdleNodes change debugging (at level 5). These changes should not be very frequent AFAICT.

Reply:
Ok.